README.md

```diff
@@ -443,33 +443,34 @@ of the time a higher-level object will suffice as an argument.
 **Bad:**
 
 ```php
-function createMenu(string $title, string $body, string $buttonText, bool $cancellable): void
+function createMenu(string $title = '', string $body = '', string $buttonText = '', bool $cancellable = false): void
 {
     // ...
 }
+
+createMenu('Foo', '', '', true);
 ```
 
 **Good:**
 
 ```php
 class MenuConfig
 {
-    public $title;
-    public $body;
-    public $buttonText;
+    public $title = '';
```

Comment:
THose by default are null, I don't see any benefit of setting them as a string :)